examples/CSIC/CSIC_SPEI_2_8.js

```diff
@@ -0,0 +1,24 @@
+
+// Retrieve the last date from the SPEI dataset
+var dataset = ee.ImageCollection("CSIC/SPEI/2_8").
+  filterDate('2021-12-01', '2022-01-01');
+
+// Select the 24-month analysis
+var spei24 = dataset.select('SPEI_24_month');
+
+// Set the visualization ranges and color palette
+var vis = {
+  min: -2.33,
+  max:  2.33,
+  palette: [
+    "#8B1A1A","#DE2929", "#F3641D",
```

Comment:
- Remove `#`
- Use single quotes `"` → `'`
- Space after `,` before next character like `', '`
- All alpha characters should be lower case, e.g. `'de2929',`